data/json/recipes/other/materials.json

```diff
@@ -144,5 +144,75 @@
     "charges": 3,
     "batch_time_factors": [ 30, 2 ],
     "components": [ [ [ "rolling_paper", 1 ] ], [ [ "chem_black_powder", 3 ], [ "chem_match_head_powder", 3 ], [ "gunpowder", 3 ] ] ]
+  },
+  {
+    "result": "bowstring_makeshift",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "survival",
+    "autolearn": true,
+    "time": "45 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "cordage", 3, "LIST" ] ] ]
+  },
+  {
+    "result": "bowstring_advanced",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "survival",
+    "skills_required": [ "archery", 2 ],
+    "difficulty": 2,
+    "book_learn": [
+      [ "recipe_bows", 1 ],
+      [ "mag_archery", 1 ],
+      [ "manual_archery", 1 ],
+      [ "book_archery", 1 ],
+      [ "mag_dude", 1 ],
+      [ "mag_survival", 1 ],
+      [ "pocket_survival", 1 ],
+      [ "survival_book", 1 ],
+      [ "textbook_survival", 1 ]
+    ],
+    "autolearn": false,
+    "time": "120 m",
+    "tools": [ [ [ "soldering_iron", 12 ], [ "lighter", 12 ], [ "ref_lighter", 12 ], [ "matches", 12 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "cordage_36", 3 ], [ "sinew", 900 ] ],
+      [ [ "wax_product", 1, "LIST" ] ],
+      [ [ "thread", 300 ], [ "sinew", 300 ], [ "plant_fibre", 300 ] ]
+    ]
+  },
+  {
+    "result": "bowstring_advanced",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "id_suffix": "with candle & flame",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "survival",
+    "skills_required": [ "archery", 2 ],
+    "difficulty": 2,
+    "book_learn": [
+      [ "recipe_bows", 1 ],
+      [ "mag_archery", 1 ],
+      [ "manual_archery", 1 ],
+      [ "book_archery", 1 ],
+      [ "mag_dude", 1 ],
+      [ "mag_survival", 1 ],
+      [ "pocket_survival", 1 ],
+      [ "survival_book", 1 ],
+      [ "textbook_survival", 1 ]
+    ],
+    "autolearn": false,
+    "time": "120 m",
+    "tools": [ [ [ "fire", -1 ] ], [ [ "candle", 12 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "cordage_36", 3 ], [ "sinew", 900 ] ],
+      [ [ "wax_product", 1, "LIST" ] ],
```

Comment:
Candle would be listed twice - through `wax_product` requirement and `tools` section.

Reply:
That's by design. You or may not be able to use the same candle that you're effectively disassembling and using for wax as a makeshift lighter -- and it doesn't appear that it's possible to use candle charges as a component, given that candles (unlike matches) aren't stored in a stack of charges. I'm not sure how best to represent this.

Reply:
I'm not sure why do we need exactly the candle (as a tool).

Reply:
I'm still learning about a lot of this stuff, which is why this is marked both by [WIP] and "Naive first draft". I don't think it should be actually pulled into the game until I've had time to internalize some of the expertise of my consultants and implement a lot more infrastructure. Everything so everything so far reflects my very narrow understanding.

In said narrow understanding, you need a small flame specifically in order to seal the knots reinforcing the bowstring's attachment points and the one over which the arrow is held without burning and thereby weakening the bowstring's cord itself. This assumption may be incorrect, and is one of the many, many things I intend to run past the people who know about this stuff.

Reply:
I think this is one of these cases where we could use abstraction. I.e. survivor is able to properly "apply" large fire from a campfire even for fine works.

Reply:
Sorry, I've gotten massively distracted by other projects. I need to review this PR to make sure it's in line with archery nerfs and update screenshots. Re: this specific input, I'm still inclined to disagree but I'll cave to seniority for now and make this change in the morning unless someone else pipes up.